io/config_loader.cpp

```diff
@@ -85,4 +92,109 @@ bool ConfigLoader::checkAndGetString(const string& parameter, string& parameter_
         logger.print(pcl::console::L_INFO, "[config_loader.cpp] INFO: Parameter %s = %s\n", parameter.c_str(), parameter_string.c_str());
         return true;
     }
+}
+
```

Comment:
See my comments for the wide_baseline_tracker_test.cpp

Reply:
The changes that resolve this case are in the commit 429d944b5468eff2331c6fa37d37ee593979f774